filters/openpolicyagent/opaauthorizerequest/opaauthorizerequest_test.go

```diff
@@ -392,6 +394,385 @@ func TestAuthorizeRequestFilter(t *testing.T) {
 			backendHeaders:    make(http.Header),
 			removeHeaders:     make(http.Header),
 		},
+		// Testing same cases with eopa plugins enabled
```

Comment:
Same as other, this is duplicating lot of lines. 
Shall we introduce a nested loop that iterate over eopa enabling and then iterate over each scenario?

This will provide a structure that can scale for different combination of configuration option we support.